src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -1,95 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
```

Comment:
Can we remove the old integration test project entirely at this point?

Reply:
Yes I'll do that in the coming week. There is a bunch of support code in RunTests and the build scripts as well.